packages/nelua/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+git clone https://github.com/edubart/nelua-lang
```

Comment:
pin an exact version with `git checkout`

Reply:
that's not possible yet, because there's no release, i'll close the pr for now.

Reply:
You could pin an exact commit